src/components/ui/Table/linksColumns.tsx

```diff
@@ -146,6 +136,87 @@ export default function ProxiedPathRow({ item }: ProxiedPathRowProps) {
           proxiedPath={item}
         />
       ) : null}
-    </>
+    </div>
   );
 }
+
+const tooltipTriggerClasses = 'max-w-full truncate';
+
+export const linksColumns: ColumnDef<ProxiedPath>[] = [
+  {
+    accessorKey: 'sharing_name',
+    header: 'Name',
+    cell: ({ row }) => {
+      const item = row.original;
+      return (
+        <div className="min-w-0 max-w-full">
+          <FgTooltip
+            label={item.sharing_name}
+            triggerClasses={tooltipTriggerClasses}
+          >
+            <Typography className="text-foreground truncate">
+              {item.sharing_name}
+            </Typography>
+          </FgTooltip>
+        </div>
+      );
+    },
+    enableSorting: true,
+    enableColumnFilter: false
```

Comment:
I noticed that you didn't have to set the `enableColumnFilter: false` on the jobsColumns object. Does the false have to be set here, because you set one of the columns to true?